### DISCUSSION TITLE
Variation in EF sum - Passing flight object as a list

### DISCUSSION THREAD

**Discussion by nataj27** [contrailcirrus/pycontrails#77] [Q&A]
I've been experimenting with the pycontrails using some sample waypoints. 
I observed that the sum of EF changes depending on the order in which flights are stored in the flight object list. 

For Example: If the flight object is in the following order -->Flight_object_list = [FL1_obj, FL2_obj, FL3_obj, FL4_obj] the sum of EF for each of the flights are different compared to when the Flight_object_list = [FL3_obj, FL4_obj, FL1_obj, FL2_obj].

**Comment by mlshapiro** [comment]
@nataj27 Thanks for starting the discussion. This sounds like a bug so I'll transfer to an issue.

Can you attach a script or notebook where you see this? @zebengberg perhaps you can confirm this is reproducible?

**Reply by mlshapiro** [reply]
Issue created in #78

**Reply by mlshapiro** [reply]
Resolved by @nataj27
